assets/src/blocks/ShareButtons/EmailShare/EmailShareFrontend.js

```diff
@@ -0,0 +1,32 @@
+import { SvgIcon } from "../SvgIcon";
+import { parseUtmParams } from '../utils';
+
+const { __ } = wp.i18n;
+
+export const EmailShareFrontend = ({
```

Comment:
All these provider specific components are mostly the same code, except for a few props (`href`, `className`, `SvgIcon.name`, and the screen reader text).

Wouldn't it be better to create a single `ShareButton` component?
```js
<ShareButton
  // ...paramsThatAreSameForAll,
  href={`mailto:...`}
  className='email'
  icon='envelope'
>

Reply:
It's a really good point!

Reply:
It was implemented separately since I initially thought to develop each as a block, but now it doesn't make sense!